src/components/Scheme/ContributionRewardExtRewarders/Competition/List.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from "react";
```

Comment:
again would prefer to name this file CompetitionList. also good for filename to match component name

Reply:
This is one of the files that must confirm to a naming convention.  (I explained in an earlier response)

Reply:
It might be possible to define the convention such that the containing folder name must be prepended to the file name.

Reply:
that would be ideal to me. but not a huge deal